initfiles/componentfiles/config2xml/environment190.xml

```diff
@@ -0,0 +1,6093 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Edited with ConfigMgr on ip  on 2018-03-23T18:29:29 -->
```

Comment:
This file seems out of place in this folder, it prob shouldn't be checked in at all